lib/model/CAnomalyScore.cc

```diff
@@ -271,23 +279,19 @@ bool CAnomalyScore::CNormalizer::canNormalize() const {
 
 bool CAnomalyScore::CNormalizer::normalize(const CMaximumScoreScope& scope,
                                            double& score) const {
-
     if (score == 0.0) {
         // Nothing to do.
         return true;
     }
-
     if (m_RawScoreQuantileSummary.n() == 0) {
-        LOG_ERROR(<< "No scores have been added to the quantile summary");
-        return false;
+        score = 0.0;
+        return true;
     }
```

Comment:
Are we intentionally going through this code branch now?  I would have thought it's a coding error to ask to normalize a score when we've never generated any score.

Reply:
We don't update the quantiles with every score any more for short bucket lengths. So at the very beginning we hit the situation we haven't yet added any scores. I think this is harmless: it is only the first 30 mins of modelling, in practice scores always work out as zero at this point anyway. I don't think it is worthwhile trying to detect that we're not in start up, so I just don't bother to error in this case anymore.